first commit for python library announcement (#112)
spokestack/spokestack-website: Introducing the Spokestack Python Library/index.md

---
title: Spokestack Python Library

> date: '2020-010-06'

Reviewer: ```suggestion
date: '2020-10-06'
```

> description: Launching the Spokestack Python library that brings the Spokestack speech pipeline to RaspberryPi, Tinker Board, and PCs.

Reviewer: ```suggestion
description: Launching the Spokestack Python library that brings the Spokestack speech pipeline to Raspberry Pi, Tinker Board, and PCs.
```

tags: Engineering
author: mike
draft: false
---

> Spokestack's tagline, "Give your app a voice interface," has mainly focused on mobile apps since we launched in January. Today we're expanding the type of apps we can voice-enable to almost any project with the launch of [Spokestack Python](https://github.com/spokestack/spokestack-python). Want to try it out now? [Will Rice](https://twitter.com/_Will_Rice), who built the library, has written a tutorial **(LINK NEEDED)** for the "Minecraft Helper" example we developed for our [mobile platforms](https://www.spokestack.io/blog/porting-a-smart-speaker-voice-app-to-mobile-part-1).

Reviewer: ```suggestion
Spokestack's tagline, "Give your app a voice interface," has mainly focused on mobile apps since we launched in January. Today we're expanding the type of apps we can voice-enable to almost any project with the launch of [Spokestack Python](https://github.com/spokestack/spokestack-python). Want to try it out now? [Will Rice](https://twitter.com/_Will_Rice), who built the library, has written a [tutorial](/blog/porting-the-alexa-minecraft-skill-to-python-using-spokestack) for the "Minecraft Helper" example we developed for our [mobile platforms](/blog/porting-a-smart-speaker-voice-app-to-mobile-part-1).
```

> Just as they do with our [iOS](https://github.com/spokestack/spokestack-ios), [Android](https://github.com/spokestack/spokestack-android), and [React Native voice libraries](https://github.com/spokestack/react-native-spokestack), developers can use Spokestack as a complete speech pipeline solution to talk to their customers. All you need to do to get started is [create a Spokestack account](https://www.spokestack.io/create) and get your API credentials. The same API that powers our mobile libraries also powers Spokestack Python. We want developers to be able to write once, talk everywhere using Spokestack.

Reviewer: ```suggestion
Just as they do with our [iOS](https://github.com/spokestack/spokestack-ios), [Android](https://github.com/spokestack/spokestack-android), and [React Native voice libraries](https://github.com/spokestack/react-native-spokestack), developers can use Spokestack as a complete speech pipeline solution to talk to their customers. All you need to do to get started is [create a Spokestack account](/create) and get your API credentials. The same API that powers our mobile libraries also powers Spokestack Python. We want developers to be able to write once, talk everywhere using Spokestack.
```

![speech_pipeline.png](./speech_pipeline.png)

You have everything you need to add a voice interface to any project.

> As with all of our libraries, developers can create a [custom wakeword](https://www.spokestack.io/blog/wakewords-for-mobile-apps) and a text-to-speech voice that is unique to their brand and application. With the Spokestack Python library, the voice interface is less tied to a platform, opening up a number of application opportunities.

Reviewer: ```suggestion
As with all of our libraries, developers can create a [custom wakeword](/blog/wakewords-for-mobile-apps) and a text-to-speech voice that is unique to their brand and application. With the Spokestack Python library, the voice interface is less tied to a platform, opening up a number of application opportunities.
```

> Got an Alexa or Google Assistant app you want to set up as a kiosk? Now you can build one on a low-energy, small-form device such as a Raspberry Pi or [Tinker Board](https://www.asus.com/us/Single-Board-Computer/Tinker-Board/). Maybe you want to set up something on an old PC with a mic and a speaker. Simply [export your NLU model](https://www.spokestack.io/docs/Concepts/export), and you can do it!

Reviewer: ```suggestion
Got an Alexa or Google Assistant app you want to set up as a kiosk? Now you can build one on a low-energy, small-form device such as a Raspberry Pi or [Tinker Board](https://www.asus.com/us/Single-Board-Computer/Tinker-Board/). Maybe you want to set up something on an old PC with a mic and a speaker. Simply [export your NLU model](/docs/Concepts/export), and you can do it!
```
